Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -159,7 +127,7 @@ function Validate-AzurePowershellVersion
 
     if(!$versionCompatible)
     {
-        Write-TaskSpecificTelemetry "PREREQ_UnsupportedAzurePSVersion"
+        Write-Telemetry "Task_InternalError" "UnsupportedAzurePSVersion"
```

Comment:
yes, as in this case we are saying "UnsupportedAzurePSVersion" some other like - ClassicStorageAccountNotFound, StorageAccountNotFound